engine/src/test/java/io/camunda/zeebe/engine/processing/deployment/DeploymentRejectionTest.java

```diff
@@ -281,9 +281,9 @@ public void shouldDoAtomicDeployments() {
 
     // then
     assertThat(
-        RecordingExporter.records()
-            .limit(r -> r.getRejectionType() == RejectionType.INVALID_ARGUMENT)
-            .collect(Collectors.toList()))
+          RecordingExporter.records()
+              .limit(r -> r.getRejectionType() == RejectionType.INVALID_ARGUMENT)
+              .collect(Collectors.toList()))
```

Comment:
💭 Which IDE are you using? If you're using IntelliJ-IDEA, the Save Actions plugin should've been set up automatically to apply our code format as soon as you save the file. This way, you don't have to remember to apply our formatting. Let me know if you want to look at this together.